pallets/xcm-helper/src/benchmarking.rs

```diff
@@ -0,0 +1,77 @@
+use super::*;
+
+#[allow(unused)]
+use crate::Pallet as XcmHelper;
+use frame_benchmarking::{account, benchmarks, whitelisted_caller, Box};
+use frame_support::{
+	sp_runtime::{traits::Hash, SaturatedConversion},
+	traits::{fungibles::Mutate, Currency},
+	BoundedVec,
+};
+use frame_system::RawOrigin;
+use parity_scale_codec::Encode;
+use sp_core::{
+	ecdsa::{self, Signature},
+	ConstU32,
+};
+use sp_std::vec;
+use xcm::{
+	latest::{
+		Error as XcmError, Fungibility, Junction, Junctions, MultiAsset, MultiAssets,
+		MultiLocation, NetworkId,
+	},
+	v1::AssetId,
+	v2::WeightLimit,
+	VersionedMultiAssets, VersionedMultiLocation,
+};
+
+const SEED: u32 = 0;
+
+benchmarks! {
+	set_thea_key {
+		let b in 1 .. 1000;
+		let public_key = sp_core::ecdsa::Public::from_raw([3, 31, 175, 212, 203, 148, 135, 36, 88, 104, 149, 228, 133, 42, 19, 172, 28, 211, 198, 250, 241, 19, 167, 9, 117, 60, 3, 83, 155, 157, 81, 94, 70]);
+	}: _(RawOrigin::Root, public_key)
+
+	create_parachain_asset {
+		let b in 1 .. 1000;
+		let asset_location = MultiLocation::new(1, Junctions::X1(Junction::Parachain(2011)));
+		let asset_id = AssetId::Concrete(asset_location);
+		let account: T::AccountId = account("mem1", b, SEED);
+	}: _(RawOrigin::Signed(account), Box::new(asset_id))
+
+	whitelist_token {
+		let b in 1 .. 1000;
+		let token = b as u128;
+		let account: T::AccountId = account("mem1", b, SEED);
+	}: _(RawOrigin::Signed(account), token)
+
+	withdraw_asset {
+		let b in 1 .. 1000;
+		let asset_location = MultiLocation::new(1, Junctions::X1(Junction::Parachain(0)));
+		let asset_id = AssetId::Concrete(asset_location);
+		let multi_asset = MultiAsset::from((asset_id, Fungibility::Fungible(1000000000)));
+		let wrapped_multi_asset = VersionedMultiAssets::V1(MultiAssets::from(multi_asset));
+		let boxed_asset_id = Box::new(wrapped_multi_asset);
+		let destination = MultiLocation::new(1, Junctions::X2(Junction::Parachain(2011), Junction::AccountId32 { network: NetworkId::Any, id: [1;32] }));
+		let boxed_multilocation = Box::new(VersionedMultiLocation::V1(destination));
+		let boundec_vec:  BoundedVec<
```

Comment:
max size of this vector is 10 so populate it to max capacity for worst case benchmarking.